app/src/main/java/net/gsantner/markor/frontend/MarkorDialogFactory.java

```diff
@@ -47,6 +50,7 @@
 import net.gsantner.markor.frontend.textview.TextViewUtils;
 import net.gsantner.markor.model.AppSettings;
 import net.gsantner.opoc.frontend.GsSearchOrCustomTextDialog;
+import net.gsantner.opoc.frontend.GsSearchOrCustomTextDialog.DialogOptions;
```

Comment:
As we use this class in dozens of places in this file, I imported it directly in order to make the code look cleaner